### ISSUE TITLE
change to frontpage to be /repositories

### ISSUE THREAD

**Issue by goern** [kwk/docker-registry-frontend#34]
It seems like there is one additional click all the time.. from / to /registry or is this configurable by now?

**Comment by kwk** [comment]
No, this is not configurable (yet) but don't hesitate to code something and create a PR. We can discus it then. The correct suffix is `#/repositories` not `/registry`. This hashtag style URLs sucks with SEO and is something I would love to get rid of. This might be a starting point https://www.google.com/search?q=angular+hashtag+url

**Comment by HadesArchitect** [comment]
I've solved this issue by adding a 'RewriteRule ^/?$ /repositories/ [L,R]' to /etc/apache2/sites-enabled/docker-site.conf 

It's a workaround, of course, but easy to implement.
